src/Sylius/Behat/Page/Admin/Product/FormTrait.php

```diff
@@ -220,6 +223,45 @@ private function selectAttributeToBeAdded(string $attributeName): void
         );
     }
 
+    /*
+     * Associations management
```

Comment:
Instead of tagging these methods, they could be extracted into a separate trait, and ideally, instead of a trait, a form element could be implemented

Reply:
I wanted to separate this part to form element as you suggested, but in this case the better option is to remove this trait and build it from scratch on form elements, we have some places with traits, maybe we could do unification of this in separate PR in all places? 🤔